module4/labs/Week2/IntroToAPIs.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title:  Intro to APIs Lab
+---
+

```

Comment:
Great exercises!  I think we could also include an instructor note that if students finish early, they could try making some of these requests and working with the resulting data from a console application as well.

Reply:
I like this idea! It's so hard to know how long a lesson will be, always good to have backup 🙃